packages/trpc/server/createContext.ts

```diff
@@ -37,7 +36,6 @@ export type CreateInnerContextOptions = {
         selectedCalendars?: Partial<SelectedCalendar>[];
         rawAvatar?: string;
       };
-  i18n?: Awaited<ReturnType<typeof serverSideTranslations>>;
```

Comment:
I didn't found anywhere we're using this. And I think this is too heavy for TS inferring.